docs/kb/semgrep-supply-chain/why-no-findings.md

```diff
@@ -0,0 +1,89 @@
+---
+description: Troubleshoot why findings for Semgrep Supply Chain are not showing.
+tags:
+  - Semgrep Supply Chain
+---
+# Why aren't Supply Chain findings showing?
+
+## Ensure compatibility
+
+First, verify that your repository meets the basic requirements for Semgrep Supply Chain:
+
+### Are the language and ecosystem supported?
+
+Check the [Supported Languages table](/docs/supported-languages/#general-availability) for support for the project's language and ecosystem, as well as any ecosystem-specific requirements.
+
+###  Is the supported lockfile present in an appropriate location with the expected name?
+
+Semgrep Supply Chain searches the parent directories of any files for a language for the nearest relevant lockfile. This typically means that monolithic repositories (monorepos) are recognized by the presence of the lockfiles in subdirectories.
+
+Semgrep Supply Chain only recognizes the lockfile names indicated in the [Supported Languages table](/docs/supported-languages/#general-availability).
+
+### Does the lockfile contain dependencies with exact versions?
+
+If your dependency file is a [manifest file](/docs/semgrep-supply-chain/glossary/#manifest-file) and does not specify exact (pinned) versions for all dependencies, Semgrep Supply Chain does not report vulnerabilities for the dependencies that are not pinned. An unpinned dependency may already be installed at a safe version for a particular [Advisory](https://semgrep.dev/docs/semgrep-supply-chain/glossary/#advisory).
```

Comment:
For "An unpinned dependency may already be installed at a safe version for a particular Advisory", what do we mean by this? Do we mean that this is why we don't check for vulns for unpinned dependencies?

Reply:
yep! do you have a suggestion to make that more clear?